File: PHP/7php 3/twoArePositive.php
<?php
/*
|--------------------------------------------------------------------------
| Two numbers are positive
|--------------------------------------------------------------------------
|
*/

function twoArePositive(...$arr) {
   return count(array_filter($arr, fn($i) => $i > 0)) == 2;
}


function twoArePositive(int $a, int $b, int $c): bool {
   return ($a > 0) + ($b > 0) + ($c > 0) === 2;
}


function twoArePositive(int $a, int $b, int $c): bool {
   return count(array_filter([$a,$b,$c],fn($n)=>$n>0))==2;
}


print_r(twoArePositive(2, 4, -3) . ' ' . true . '
');
print_r(twoArePositive(-4, 6, 8) . ' ' . true . '
');
print_r(twoArePositive(4, -6, 9) . ' ' . true . '
');
print_r(twoArePositive(-4, 6, 0) . ' ' . false . '
');
print_r(twoArePositive(4, 6, 10) . ' ' . false . '
');
print_r(twoArePositive(-14, -3, -4) . ' ' . false . '
');

File: Python/7py 3/two_are_positive.py
"""
|--------------------------------------------------------------------------
| Two numbers are positive
|--------------------------------------------------------------------------
|
"""

def two_are_positive(*args):
   return len([i for i in args if i > 0]) == 2


def two_are_positive(a, b, c):
   return sum([a>0, b>0, c>0]) == 2


def two_are_positive(*args):
   return sum(x > 0 for x in args) == 2


print(two_are_positive(2, 4, -3), True)
print(two_are_positive(-4, 6, 8), True)
print(two_are_positive(4, -6, 9), True)
print(two_are_positive(-4, 6, 0), False)
print(two_are_positive(4, 6, 10), False)
print(two_are_positive(-14, -3, -4), False)